compiler/rustc_mir_transform/src/lower_slice_len.rs

```diff
@@ -69,7 +69,7 @@ fn lower_slice_len_call<'tcx>(
                     // perform modifications
                     // from something like `_5 = core::slice::<impl [u8]>::len(move _6) -> bb1`
                     // into `_5 = Len(*_6)
-                    // goto bb1
+                    // goto bb1`
```

Comment:
I don't know what the correct change is here, but I'm pretty sure this isn't it. This implies that the newline after the assignment isn't meaningful, and it is. I would use triple backticks or just none at all, such as

```
                    // from something like
                    // _5 = core::slice::<impl [u8]>::len(move _6) -> bb1
                    // into 
                    // _5 = Len(*_6)
                    // goto bb1
```